finetuner/tuner/metrics.py

```diff
@@ -0,0 +1,162 @@
+from typing import Optional, Dict, List, Tuple
```

Comment:
this file is mostly redundant, as it is basically doing 3 lines of code:

```python
import jina.math.evaluations
rel = _doc_to_relevance(doc)[0]
getattr(jina.math.evaluation, metric_str)(rel, *args)
```

Reply:
@hanxiao are you ok with the current format? Or should I discard `metrics.py` and go with the following:

```
import jina.math.evaluations
rel, max_rel = _doc_to_relevance(doc)
if metric_str == 'recall' or metric_str == 'f1score':
    return getattr(jina.math.evaluation, metric_str)(rel, max_rel, *args)
return getattr(jina.math.evaluation, metric_str)(rel, *args)
```

Reply:
If these four lines fully functionally replace this file, let's replace it.

Reply:
Removed wrapping logic